test/utils/bindActionCreators.spec.js

```diff
@@ -37,4 +37,18 @@ describe('bindActionCreators', () => {
       { id: 1, text: 'Hello' }
     ]);
   });
+
+  it('should throw an invariant violation for an undefined actionCreator', () => {
+    expect(() => {
+      bindActionCreators(undefined, store.dispatch);
+    }).toThrow('bindActionCreators expected an object or a function, instead received undefined. ' +
```

Comment:
For consistency, let's test for `null` too (I'm not sure it's a real use case, but passing `null` never makes sense).